emails/utils.py

```diff
@@ -22,6 +28,11 @@ def func_wrapper(*args, **kwargs):
     return timing_decorator
 
 
+def incr_if_enabled(name, value):
```

Comment:
```suggestion
def incr_if_enabled(name, value=1):
```
Default value of `1`? It seems like everytime the `incr_if_enabled` increments by 1 only.